.idea/gradle.xml

```diff
@@ -14,13 +14,7 @@
             <option value="$PROJECT_DIR$/example_app" />
           </set>
         </option>
-        <option name="myModules">
```

Comment:
At some point we should remove most of the files in `.idea` from the repo.  I recently used this article as a guide for doing so on a work project: https://goo.gl/HPnSVd